src/Plaid/Management/PlaidErrorCode.cs

```diff
@@ -0,0 +1,306 @@
+﻿using System.ComponentModel;
```

Comment:
File should be removed in favor of existing `ErrorCode` enum.

Reply:
forgot I had those old ones still in there... will remove